wiki/Reporting_Bad_Behaviour/id.md

```diff
@@ -1,53 +1,58 @@
----
-outdated: true
-outdated_since: 83725ce048670f4ac909cf76e228a9a5b792df4a
----
-
-# Melaporkan Perilaku Buruk
+# Melaporkan perilaku buruk
 
 ## Apa itu sistem lapor?
 
 Sistem lapor adalah sebuah fitur penting untuk memberitahu moderator mengenai pemain yang berperilaku buruk untuk diberikan hukuman yang sesuai.
 
-Tidak hanya sebatas obrolan dalam game tetapi juga di tempat lain seperti forum dan wiki itu sendiri.
+Tidak hanya sebatas obrolan di dalam gim saja tetapi juga di tempat lain seperti forum dan wiki itu sendiri.
 
 ## Bagaimana cara melaporkan seseorang
 
-### Menggunakan perintah !report dalam game
+### Menggunakan perintah !report di dalam gim
 
-Penggunaan: `!report user_name [alasan]`
+Penggunaan: `!report nama_pengguna [alasan]`
 
 Parameter **[** dan **]** merupakan opsional.
 
-Jika pengguna memiliki spasi di nama pengguna mereka, spasi harus diganti dengan garis bawah_ karena obrolan dalam game osu! didasarkan pada [IRC](/wiki/Internet_Relay_Chat "Internet Relay Chat (IRC)").
+Jika pengguna memiliki spasi di nama pengguna mereka, spasi harus diganti dengan garis bawah `_` karena obrolan di dalam gim osu! didasarkan pada [IRC](/wiki/Internet_Relay_Chat "Internet Relay Chat").
 
-`!report This User Name is spamming in #osu` menjadi `!report This_User_Name is spamming in #osu`
+`!report Nama Pengguna Ini is spamming in #osu` menjadi `!report Nama_Pengguna_Ini is spamming in #osu`
```

Comment:
```suggestion
`!report Nama Pengguna Ini melakukan tindakan spamming di #osu` menjadi `!report Nama_Pengguna_Ini melakukan tindakan spamming di #osu`
```

Reply:
alasan ada baiknya ditulis tetap pake bahasa Inggris, karena GMT atau Chat Moderator belum tentu online 24 jam. Jadi moderator lain bisa bantu buat menanganinya. Mungkin @Nivalyx bisa minta tolong pendapatnya soal pengisian `[alasan]` di fitur `!report`?

Reply:
imo memang lebih baik dibiarin dalam bahasa inggris aja sih. hampir semua laporan yang diterima masuk sama gmt itu ditulis dalam bahasa inggris soalnya, karena memang sebagaimana yang tertulis di http://osu.ppy.sh/wiki/fi/People/The_Team/Global_Moderation_Team "All GMT members speak English (...) unless noted otherwise"

di samping itu juga (iirc) kalau kita ngelapor orang lain pakai feature "Report User" in-game -- terlepas dari bahasa apapun yang lagi dipasang di client osu!-nya -- laporan yang diterima bakal tetap dalam bahasa inggris

![image](https://user-images.githubusercontent.com/36564236/107674357-78739880-6cc9-11eb-9e16-036dd42e2669.png)

Reply:
okthen, keep. thankyou Niva and SeverinoAji!